spec/index.html

```diff
@@ -543,76 +543,66 @@ <h3>Algorithm</h3>
         <li id="ca-3">Create a list of non-normalized <a>blank node identifiers</a>
           <var>non-normalized identifiers</var> and populate it using the keys
           from the <a>blank node to quads map</a>.</li>
-        <li id="ca-4">Initialize <var>simple</var>, a boolean flag, to
-          <code>true</code>.</li>
-        <li id="ca-5">While <var>simple</var> is <code>true</code>,
-          issue canonical identifiers for <a>blank nodes</a>:
+        <li id="ca-4">Clear <a>hash to blank nodes map</a>.</li>
```

Comment:
Now we are not in the while loop, it might be better to use "Create" just like the other parts in this document.
```suggestion
        <li id="ca-4">Create <a>hash to blank nodes map</a>.</li>
```

Reply:
The **hash to blank nodes map** is part of the canonicalization state, so _creating_ it is probably not the right term to use here, either.

Given that step 1 initializes the canonicalization state, which implicitly describes the hash to blank nodes map, I think that step 4 becomes unnecessary, but can be described in a summary description on step 1 for clarity.

Reply:
@yamdan Please see if the updated algorithm description suites.

Reply:
Thank you @gkellogg, I agree with your updated description; I had overlooked that the canonicalization state in step 1 includes the hash to blank nodes map.